predictsignauxfaibles/__main__.py

```diff
@@ -18,11 +18,23 @@
 # Mute logs from sklean_pandas
 logging.getLogger("sklearn_pandas").setLevel(logging.WARNING)
 
+args_to_attrs = {
+    "train_spl_size": ("train", "sample_size"),
+    "test_spl_size": ("test", "sample_size"),
+    "predict_spl_size": ("predict", "sample_size"),
+    "train_proportion_positive_class": ("train", "proportion_positive_class"),
+    "train_from": ("train", "date_min"),
+    "train_to": ("train", "date_max"),
+    "test_from": ("test", "date_min"),
+    "test_to": ("test", "date_max"),
+    "predict_on": ("predict", "date_min"),
+}
```

Comment:
```suggestion
ARGS_TO_ATTRS = {
    "train_spl_size": ("train", "sample_size"),
    "test_spl_size": ("test", "sample_size"),
    "predict_spl_size": ("predict", "sample_size"),
    "train_proportion_positive_class": ("train", "proportion_positive_class"),
    "train_from": ("train", "date_min"),
    "train_to": ("train", "date_max"),
    "test_from": ("test", "date_min"),
    "test_to": ("test", "date_max"),
    "predict_on": ("predict", "date_min"),
}
```

since this is a constant

let's write a test `test_args_to_attrs_consistency` that make sure that we do not change the name of the args in the CLI while forgetting to change this